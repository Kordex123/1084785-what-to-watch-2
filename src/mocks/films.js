const films = [
  {
    id: 10111,
    genre: `Comedies`,
    starring: [],
    movieTitle: `Fantastic Beasts: The Crimes of Grindelwald`,
    previewMovieImage: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    previewVideoLink: `http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerMeltdowns.mp4`,
  },
  {
    id: 20222,
    genre: `Comedies`,
    movieTitle: `Bohemian Rhapsody`,
    previewMovieImage: `bohemian-rhapsody.jpg`,
    previewVideoLink: `http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerFun.mp4`,
  },
  {
    id: 30333,
    genre: `Documentary`,
    movieTitle: `Macbeth`,
    previewMovieImage: `macbeth.jpg`,
    previewVideoLink: `https://file-examples.com/wp-content/uploads/2017/04/file_example_MP4_640_3MG.mp4`,
  },
  {
    id: 40444,
    genre: `Romance`,
    movieTitle: `Aviator`,
    previewMovieImage: `aviator.jpg`,
    previewVideoLink: `https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/360/Big_Buck_Bunny_360_10s_1MB.mp4
`,
  },
  {
    id: 50555,
    genre: `Romance`,
    movieTitle: `Pulp Fiction`,
    previewMovieImage: `pulp-fiction.jpg`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
];

export default films;
