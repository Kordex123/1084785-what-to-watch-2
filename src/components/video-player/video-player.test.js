import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from "./video-player";
import films from "../../mocks/films";

it(`VideoPlayer correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<VideoPlayer
      film={films[3]}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
