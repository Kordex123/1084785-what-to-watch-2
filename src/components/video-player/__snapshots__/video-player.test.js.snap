// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders after relaunch 1`] = `
<video
  className="player__video"
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  poster="aviator.jpg"
  type="video/mp4"
>
  <source
    src="https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/360/Big_Buck_Bunny_360_10s_1MB.mp4
"
  />
</video>
`;
